docs/advanced.md

```diff
@@ -262,9 +262,10 @@ client = httpx.Client(trust_env=False)
 
 ## HTTP Proxying
 
-HTTPX supports setting up [HTTP proxies](https://en.wikipedia.org/wiki/Proxy_server#Web_proxy_servers) via the `proxies` parameter to be passed on client initialization.
+HTTPX supports setting up [HTTP proxies](https://en.wikipedia.org/wiki/Proxy_server#Web_proxy_servers) via the `proxies` parameter to be passed on client initialization or top-level API functions like `httpx.get(..., proxies=...)`.
 
 _Note: SOCKS proxies are not supported yet._
+_Note: `proxies` parameter passed to `Client.request` is not supported yet._
```

Comment:
I don't think we need this note, especially since we most likely won't support this at all in the foreseeable future (it's a consequence of our tighter transport API).